recipes/marisa-trie/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "marisa-trie" %}
```

Comment:
If we are going to provide the C++ library as a separate shared library (because we will be building R bindings or something) then we definitely should prefix it.